code/modules/surgery/organs/lavaland.dm

```diff
@@ -61,6 +61,12 @@
 		if(owner.mind.assigned_role == "Mime")
 			power_multiplier *= 0.5
 
+	//WGW
+	else if(findtext(command, "one day, while andy") || findtext(command, "one day while andy"))
```

Comment:
Change the else if to just an if.  Right now this will only work on mindless mobs.  You'll also want to change the next if to an else if.